lms/djangoapps/instructor/views/api.py

```diff
@@ -1138,6 +1138,10 @@ def get_students_features(request, course_id, csv=False):  # pylint: disable=red
         query_features.append('cohort')
         query_features_names['cohort'] = _('Cohort')
 
+    if course.teams_enabled:
```

Comment:
Thinking aloud, and nothing that needs to be addressed in this PR... it would be great if the reports could be pluggable so that we could inject this functionality from the Teams Django app. Maybe teams is now such a core concept that this isn't necessary, but thinking that way means that edx-platfform just keeps growing. Even if we don't separate it out into a separate repo, having this be pluggable would help keep the code more modular, and make it easier to understand and maintain. My dream would be to have all changes needed by teams live in the Teams app, but obviously there's a long way to go to achieve that.